data/tlg0014/tlg001/tlg0014.tlg001.perseus-eng2.xml

```diff
@@ -29,14 +29,19 @@
                 <biblStruct>
                     <monogr>
                         <author>Demosthenes</author>
-                        <title>Demosthenes. Vol. I. Olynthiacs, Philippics, Minor Public Speeches, Speech Against Leptines, I-XVII, XX. With an English translation by J. H. Vince.</title>
+                        <title>Demosthenes</title>
+                        <title type="sub">Olynthiacs, Philippics, Minor Public Speeches, Speech Against Leptines, I-XVII, XX</title>
+                        <editor role="translator">J.H. Vince</editor>
```

Comment:
I have been expanding the initials/

Reply:
I can do a global change at the end to do that.